src/main/resources/application.yaml

```diff
@@ -16,5 +16,11 @@ info:
     description: Ingest scanned documents.
 
 storage:
-  account_name: ${STORAGE_ACCOUNT_NAME}
-  key: ${STORAGE_KEY}
+  account_name: ${STORAGE_ACCOUNT_NAME:bulkscanning}
+  key: ${STORAGE_KEY:dGVzdA==} #Base 64 encoded
+
+accesstoken:
```

Comment:
Just a suggestion: let's keep camel case everywhere for readability. Or underscores. Or whichever format is OK with us

Reply:
spring boot throws exception if we use camel case/underscores